src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -385,6 +486,24 @@ def _parse_lttng_data(
         events_ = None if len(events) == 0 else events
         return data, events_, begin, end
 
+    @staticmethod
+    def _compare_init_event(
+        event1: dict,
+        event2: dict,
+    ) -> int:
+        if event2['_timestamp'] < event1['_timestamp']:
+            return 1
+        if event2['_timestamp'] > event1['_timestamp']:
+            return -1
+        # same timestamp
+        if Lttng._prioritized_init_events.index(event2['_name']) < \
+                Lttng._prioritized_init_events.index(event1['_name']):
+            return 1
+        if Lttng._prioritized_init_events.index(event2['_name']) > \
+                Lttng._prioritized_init_events.index(event1['_name']):
```

Comment:
I suggest using constants instend of plain strings.

```suggestion
        if event2[LttngEventFilter.TIMESTAMP] < event1[LttngEventFilter.TIMESTAMP]:
            return 1
        if event2[LttngEventFilter.TIMESTAMP] > event1[LttngEventFilter.TIMESTAMP]:
            return -1
        # same timestamp
        if Lttng._prioritized_init_events.index(event2[LttngEventFilter.NAME]) < \
                Lttng._prioritized_init_events.index(event1[LttngEventFilter.NAME]):
            return 1
        if Lttng._prioritized_init_events.index(event2[LttngEventFilter.NAME]) > \
                Lttng._prioritized_init_events.index(event1[LttngEventFilter.NAME]):
```

Reply:
Changed to use constants